example/webpack.config.js

```diff
@@ -60,7 +60,7 @@ module.exports = {
     extensions: [".js", ".ts", ".jsx", ".tsx"],
     alias: {
       "@": path.resolve(__dirname, "assets/js"),
-      // Necessary to deduplicate React due to npm link:
+      // Necessary to deduplicate React due to pnpm link:
```

Comment:
Maybe we don't need that anymore?

Reply:
I'll test

Reply:
We still need it. I got that same error when I removed it.

Reply:
It's strange... does that mean our lib needs that? That's bad for us...
Can you check in a clean project?

Reply:
We only need that when we are developing the client and using `link` to test the changes.

Reply:
It'd be a local config, not a prod

Reply:
Ah OK!